### ISSUE TITLE
Freeze with custom script using inotifywait

### ISSUE THREAD

**Issue by lahwaacz** [Alexays/Waybar#242]
I'm attempting to use the following custom module:
```
"custom/mail": {
    "format": "\uf0e0 {}",
    "exec": "cat /dev/shm/new-mails-count; while inotifywait -q -e modify /dev/shm/new-mails-count > /dev/null; do cat /dev/shm/new-mails-count; done",
    "return-type": "json",
    "tooltip": false
},
```
This seems to work by itself, but when I disable one of the two outputs (eDP-1), the whole waybar freezes even on the remaining output (HDMI-A-1). It does not happen without the custom module or when I don't use `inotifywait`:
```
    ...
    "exec": "cat /dev/shm/new-mails-count",
    "interval": 1,
    ...
```
But obviously I don't like having a script executed every second.

Program versions:
- Sway 1.0
- Waybar 0.5.0

**Comment by Alexays** [comment]
Similar to #160
Custom modules need to way to kill the launched cmd

**Comment by Alexays** [comment]
Fixed with 90d89fe9742fec690677fc6965d28b25df8fc14f
